### DISCUSSION TITLE
Try loaddata in in docker

### DISCUSSION THREAD

**Discussion by yoan34** [saleor/saleor#9252] [General]
Hi,
I'm trying to use **loaddata** to populate the database, oddly it works fine in local `python3 manage.py ladata data.json`. But with docker-compose I run `docker-compose run --rm api python3 manage.py loaddata data.json` and I get an error.<br>
`django.db.utils.IntegrityError: Problem installing fixture '/app/populatedb_data.json': Could not load product.Product(pk=61): null value in column "created" violates not-null constraint`.<br>
I don't understand how to handle this error, the **data.json** was created with `python3 manage.py dumpdata --natural-foreign --natural-primary -e contenttypes -e auth.Permission --indent 2 > dump.json`<br>
When I look at the model **product_product**, I dont see a field **created**.

**Comment by vitorecodots** [comment]
test this docker exec -it {your container app name} python manage.py loaddata  {app_django}/fixtures/{your model.json}
